tox_docker.py

```diff
@@ -54,6 +64,9 @@ def tox_runtest_pre(venv):
         conf._docker_containers.append(container)
 
         container.reload()
+        gateway_ip = "0.0.0.0"
+        if conf.docker_use_gateway and container.attrs["NetworkSettings"]["Gateway"]:
```

Comment:
Is the gateway sometimes not available? I'd love for this to be the default way to do things, rather than an option, in order to keep the interface smaller, but if it's sometimes not possible, that's a bummer.

Reply:
I will check for that if it is possible that the gateway is unavailable.

Reply:
It seems to always be set. So I will remove the parameter `docker_use_gateway`.